portfolio/src/main/webapp/script.js

```diff
@@ -30,4 +30,14 @@ function  changeTitle(){
     if (i === titles.length) {
       i = 0;
     }
+}
+
+
+
+function playGame(){
```

Comment:
nit: Please use at most 2 blank lines between function definitions.

Reply:
okay I will do that